android/app/src/web/java/app/organicmaps/location

```diff
@@ -0,0 +1 @@
+../../../../google/java/app/organicmaps/location
```

Comment:
This was the easiest way to support gms and non-gms builds without creating additional dimension in build.gradle. I don't to create complexity like "packageGoogleGmsWorldDebug" and  "packageFDroidNonGmsNonWorldDebug". Symlinks allowed me to keep "packageGoogleDebug" and  "packageFdroidDebug".